examples/v0/account_management/create_customer.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example illustrates how to create a new customer under a given
+manager account.
+
+Note by default this account will only be accessible via its parent Google Ads
+manager account.
```

Comment:
Please use the note from the [Java SOAP example](https://github.com/googleads/googleads-java-lib/blob/master/examples/adwords_axis/src/main/java/adwords/axis/v201809/accountmanagement/CreateAccount.java#L41), with a few minor tweaks:

```
Note: this example must be run using the credentials of a Google Ads manager account. By
default, the new account will only be accessible via the manager account.
```

Reply:
Done